packages/tools/addon/templates/components/cs-active-composition-panel.hbs

```diff
@@ -9,62 +9,25 @@
     }}
   </div>
 
-  <div class="cs-active-composition-panel--parameters">
-    <div class="cs-active-composition-panel--title">
-      {{!-- Change icon and type depending on overlay selected --}}
-      {{!-- If no overlay is selected, type title is "Global Attributes" --}}
-      {{!-- {{cs-humanize (cs-model-type model)}} --}}
-
-      {{cs-svg "global-attributes-icon" width="14px" height="14px"}}
-      Global Attributes
-    </div>
-  </div>
-
-  <div class="cs-active-composition-panel--flex-slot">
-    <div class="cs-active-composition-panel--scroll-wrapper">
-      <div class="cs-active-composition-panel--main" data-test-cs-active-composition-panel-main>
-        <div data-test-cs-active-composition-panel-top-fields>
-          {{#each headerSectionFields key="id" as |field|}}
-            {{#let
-              (get permissions (cs-uid field.model))
-              (eq field.model model)
-              (cs-field-editor-options-for field.model field.name)
-            as |fieldModelPermissions isPageModelField editorOptions|}}
-              <div class="cs-field-editor-section cs-field-{{dasherize field.name}}">
-                {{#unless editorOptions.hideTitle}}
-                  <div class="cs-field-editor-section--title" cs-field-editor-section-title={{dasherize field.name}}>
-                    {{cs-card-meta-caption field.model field.options.caption isPageModelField}}
-                  </div>
-                {{/unless}}
-                {{cs-field-editor
-                  content=field.model
-                  field=field.name
-                  editorOptions=editorOptions
-                  enabled=enabled
-                  permissions=fieldModelPermissions
-                  onchange=(action "validate")
-                  class="cs-active-composition-panel--field-editor"
-                  errors=(get validationErrors field.name)
-                  dataTestName=(dasherize field.name)
-                }}
-              </div>
-            {{/let}}
-          {{/each}}
-        </div>
-
-        {{#each renderedFields key="id" as |fieldMark|}}
-          {{! template-lint-disable attribute-indentation }}
+  {{#if openedFieldId}}
+    <div class="cs-active-composition-panel--parameters">
+      {{#each renderedFields key="id" as |fieldMark|}}
```

Comment:
This template file was already at or beyond the maximum acceptable complexity. The copy-paste duplication introduced by this PR pushes it way beyond understandable. So we need to factor the shared pieces out into one or more separate components.

Reply:
Any better now?

Reply:
@ef4 can you take another look at this?